bin/release.py

```diff
@@ -298,16 +298,13 @@ def release_module(args) -> int:
                 r"build( openresty)? ([\d.]+) on (amd64|arm64) WAF (ON|OFF)",
                 job["name"],
             )
-            if match is None:
+
+            if not match:
                 continue
 
-            if len(match.groups()) == 4:
-                # It is an `openresty` build
-                flavor, nginx_version, arch, waf = match.groups()
+            flavor, nginx_version, arch, waf = match.groups()
+            if flavor:
                 flavor = flavor.strip()
-            else:
-                nginx_version, arch, waf = match.groups()
-                flavor = "nginx"
```

Comment:
The default flavor name is now "" and not "nginx" anymore ? Is it the desire outcome ?